src/acl.c

```diff
@@ -708,6 +729,12 @@ void ACLAddAllowedSubcommand(user *u, unsigned long id, const char *sub) {
  *              It is possible to specify multiple patterns.
  * allkeys      Alias for ~*
  * resetkeys    Flush the list of allowed keys patterns.
+ * &<pattern>   Add a pattern of channels that can be mentioned as part of
+ *              Pub/Sub commands. For instance &* allows all the channels. The
+ *              pattern is a glob-style pattern like the one of PSUBSCRIBE.
+ *              It is possible to specify multiple patterns.
+ * allchannels              Alias for &*
+ * resetchannels            Flush the list of allowed keys patterns.
```

Comment:
Instead can I recommend "nochannels"? It removes the ambiguity of what the default is. I thought resetkeys was a bad idea.

Reply:
I can live with that, but in that case I'd like to have 'nokey' as an alias for 'resetkeys' - makes sense?

Reply:
Sure